src/Components/Tab/Tab.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './Tab.css';
+
+const Tab = ({ tabs, onclick, activeTab }) => (
```

Comment:
`onClick` - it is a bad practice to name prop fields the same as existing attributes.